web/src/commands/serve.ts

```diff
@@ -26,6 +27,7 @@ exports.handler = (argv: any) => {
 
 export async function main(argv: any): Promise<void> {
   sourceMapSupport.install();
+  await startExternalAddonPulling();
```

Comment:
Making sure I'm understanding here, we are intending to prevent startup from continuing until the poller has started and a first attempt has been made to pull the addons?

Reply:
That's the thought, I feel like it would be bad if an instance went live after a deployment but all the external addons were unavailable.  I figure its better to wait for the application to become fully initialized.

Reply:
can we do this after bugsnag and newrelic init?